packages/ui/src/cairo/ERC721Controls.svelte

```diff
@@ -79,6 +79,37 @@
   </div>
 </section>
 
+<section class="controls-section">
+  <h1>
+    <!-- svelte-ignore a11y-label-has-associated-control -->
+    <label class="flex items-center tooltip-container pr-2">
+      <span>Votes</span>
+      <span class="ml-1">
+        <input type="checkbox" bind:checked={opts.votes}>
+      </span>
+      <HelpTooltip align="right" link="https://docs.openzeppelin.com/contracts-cairo/api/erc20#ERC20VotesComponent">
```

Comment:
This change should also be made in ERC20Controls.svelte

```suggestion
      <HelpTooltip align="right" link="https://docs.openzeppelin.com/contracts-cairo/api/governance#VotesComponent">
```